src/pal/inc/unixasmmacrosarm.inc

```diff
@@ -131,10 +147,12 @@ C_FUNC(\Name\()_End):
 
         __PWTB_StackAlloc = __PWTB_TransitionBlock
 
-	// Make the stack operation compatible with GDB and LIBUNWIND
-	PROLOG_PUSH		"{r7, lr}"
+        .ifnc \pushArgRegs, DoNotPushArgRegs
```

Comment:
Broken while trying to make the stack compliant with the assumption.

You don't need to try to run this code. Will Fix Soon.